server/shared/types/termdb.ts

```diff
@@ -106,6 +106,14 @@ export type Subconditions = {
 	}
 }
 
+type ValueConversion = {
+	scaleFactor: number
+	fromUnit: string // name of unit for the original value
```

Comment:
@xzhou82, It looks like the only values for `fromUnit` and `toUnit` are `day` and `year`. Is that correct? If that's the case, you can validate the input whilst type checking like this: 
```
fromUnit: 'day' | 'year',
toUnit: 'day' | 'year'
```
This will also serves as documentation if of course those are the only acceptable values.